templates/infinispan-ephemeral.json

```diff
@@ -200,29 +304,35 @@
                     "name": "hotrod",
                     "protocol": "TCP"
                   }
-                ]
+                ],
+                "volumeMounts": [
+                  {
+                    "name": "${APPLICATION_NAME}-configuration",
+                    "mountPath": "/opt/jboss/infinispan-server/standalone/configuration/custom"
+                  }
+                ],
+                "resources": {
+                  "requests": {
+                    "cpu": "0.5",
+                    "memory": "512Mi"
```

Comment:
I don't think we let you specify requests in online, have you tried deploying these templates there?  At best i think these values will get ignored in that environment.